README.md

```diff
@@ -19,24 +26,49 @@ changes if necessary.
 Any secrets added should be documented as variables in `secrets.tf` and added
 to the `terraform/secrets.sh` script in password-store.
 
-Kubernetes is managed with [kops](https://github.com/kubernetes/kops).
-
-Locked State
-------------
-
-If the state becomes locked because a process fails:
-
- * Open the Azure portal
- * Navigate to the tcterraformstate account
- * Navigate to the tfstate container
- * Find the blob in the "Leased" state
- * Click the "..." menu and select "Break Lease".
-
-Deploying a new version of a static service
--------------------------------------------
+## Deploying a new version of a static service
 
 In `static-services.tf` each service has both a `image_tag` and `image_hash`
 field that can be updated to deploy a new version of the service. The
 `image_hash` is the `sha256 Digest` of an image. Once you update this field and
 `terraform apply`, the ec2 instance will be replaced by a new one (this does
 involve downtime!) and the eip will be hooked into the new instance.
+
+## Troubleshooting
+
+### Locked State
+
+The full error message looks like (or is a variant of):
+
+`Error: Error locking state: Error acquiring the state lock: storage: service returned error: StatusCode=409, ErrorCode=LeaseAlreadyPresent, ErrorMessage=There is already a lease present.`
+
+You will need to manually break the terraform lease to perform *any* further
+actions. The lease is attached to a terraform state blob that lives in Azure.
+
+To break the lease:
+* open the Azure portal
+* navigate to Resources
+* open tcterraformstate->Blobs->tfstate
+* click on the three-dots menu for the state-test2.tf blob, and select "Break lease"
+
+See also https://github.com/hashicorp/terraform/issues/17046#issuecomment-359597285
+
+### Packet.net
+
+#### General
+
+We build each instance from scratch in packet.net, rather than tweaking an
+existing image, although we do expect to move to images soon. The from-scratch
+process means that there are many steps that can timeout or fail, usually due
+to network issues when installing large collections of Ubuntu packages.
+
+If the process fails, it is safest to perform a `terraform destroy` followed
```

Comment:
Maybe highlight here you may use `-target` to destroy to the specific instance that failed.

Reply:
Added https://github.com/taskcluster/taskcluster-infrastructure/pull/45/commits/a6dafbf602637a12e3a6c34311aea375ba648367